Graphics/GraphicsEngineD3D12/include/pch.h

```diff
@@ -44,6 +44,10 @@
 
 #include "WinHPostface.h"
 
+#ifndef NTDDI_WIN10_FE // First defined in Win SDK 10.0.20348.0
+constexpr D3D_FEATURE_LEVEL D3D_FEATURE_LEVEL_12_2 = static_cast<D3D_FEATURE_LEVEL>(0xc200);
+#endif
```

Comment:
I have `10.0.22621.0`, and `D3D_FEATURE_LEVEL_12_2` is defined

Reply:
It is not defined in `10.0.19041.0`, so it looks like `10.0.22621.0` is where it was first defined

Reply:
It is defined in 10.0.20348.0

Reply:
10.0.20348.0

```c
...
#define NTDDI_WIN10_RS5                     0x0A000006
#define NTDDI_WIN10_19H1                    0x0A000007
#define NTDDI_WIN10_VB                      0x0A000008
#define NTDDI_WIN10_MN                      0x0A000009
#define NTDDI_WIN10_FE                      0x0A00000A

#define WDK_NTDDI_VERSION                   NTDDI_WIN10_FE
```

---

10.0.22000.0

```c
...
#define NTDDI_WIN10_19H1                    0x0A000007
#define NTDDI_WIN10_VB                      0x0A000008
#define NTDDI_WIN10_MN                      0x0A000009
#define NTDDI_WIN10_FE                      0x0A00000A
#define NTDDI_WIN10_CO                      0x0A00000B

#define WDK_NTDDI_VERSION                   NTDDI_WIN10_CO
```

---

10.0.22621.0

```c
...
#define NTDDI_WIN10_FE                      0x0A00000A
#define NTDDI_WIN10_CO                      0x0A00000B
#define NTDDI_WIN10_NI                      0x0A00000C

#define WDK_NTDDI_VERSION                   NTDDI_WIN10_NI
```

Reply:
I am not saying `D3D_FEATURE_LEVEL_12_2` is not defined in 10.0.20348.0.
I am sating it _is_ defined in 10.0.22621.0, so this will fail to build with this change as the `D3D_FEATURE_LEVEL_12_2` will be double-defined.

Reply:
So the comment is then incorrect?

Reply:
Yes